app/controllers/application_controller.rb

```diff
@@ -240,7 +240,8 @@ def after_sign_in_path_for(_user)
 
   def signed_in_url
     return user_two_factor_authentication_url unless user_fully_authenticated?
-    return account_or_verify_profile_url if profile_needs_verification?
+    return reactivate_account_url if user_needs_to_reactivate_account?
+    return url_for_pending_profile_reason if user_has_pending_profile?
```

Comment:
Is it worth trying to move away from the generic "has pending profile"? If we wanted to, we could take a small shortcut and replace the conditionals used with:

```suggestion
    return url_for_pending_profile_reason if url_for_pending_profile_reason
```

It looks like the method would still be used by some analytics, but it might be worth splitting that up to be more specific too.